lib/repl.js

```diff
@@ -1140,7 +1140,7 @@ function isRecoverableError(e, self) {
       self._inTemplateLiteral = true;
       return true;
     }
-    return /^(Unexpected end of input|Unexpected token)/.test(message);
+    return /^(Unexpected (end of input|token)|missing \) after)/.test(message);
```

Comment:
`/^(?:Unexpected (?:end of input|token)|missing \) after)/` (No need to capture groups)